CareKit/CareKit/iOS/Calendar/Paging/OCKWeekCalendarPageViewController.swift

```diff
@@ -83,7 +83,8 @@ UIPageViewController, UIPageViewControllerDataSource, UIPageViewControllerDelega
     /// The initial date displayed when the view controller is loaded.
     private let startingDate = Date()
 
-    private let store: OCKAnyStoreProtocol
+    /// The store the view controller uses for synchronization.
+    public let store: OCKAnyStoreProtocol
```

Comment:
Made this store along with the one in `OCKDailyPageViewController` public readable. The reason for this is when a developer subclasses they may want to fetch/save data to the relevant `store` in the view. If it's not public, the developer will need to access a store by other means (maybe global?) which can actually be a different store than the one in the view.

Public access allows for something similar to the `careStore` environment, but in UIKit, and preservers the behavior when OCKSynchronizedStoreManager was public readable.